myapp/app/controllers/application_controller.rb

```diff
@@ -22,4 +22,10 @@ def logged_in_user
 
     redirect_to '/login', flash: { notice: t('flash_msgs.no_login') }
   end
+
+  def admin_user?
+    return false if admin?
+
+    redirect_to '/404'
```

Comment:
If a resource is inaccessible due to insufficient authorization, it is Internet practice to return a 403 !
https://developer.mozilla.org/ja/docs/Web/HTTP/Status/403

Reply:
Thanks for the comment!
It's proper to redirect to 403 since it is an authorization issue, so I changed it to redirect to 403 in this commit[3ae442e](https://github.com/Fablic/training/pull/1575/commits/3ae442eaeb808bf32337b8f9a4d2aeafd546f6bf)
Please kindly check it~
P.S. Since there are no approaches for a normal user to get access to the admin page, maybe returning 403 will tell the malicious user where the admin page is? Is it better to return 404 here?